datadog_checks_base/datadog_checks/base/utils/db/sql.py

```diff
@@ -24,3 +36,51 @@ def compute_exec_plan_signature(normalized_json_plan):
         return None
     with_sorted_keys = json.dumps(json.loads(normalized_json_plan), sort_keys=True)
     return format(mmh3.hash64(with_sorted_keys, signed=False)[0], 'x')
+
+
+class eventEncoder(json.JSONEncoder):
```

Comment:
nit: should be CamelCase

Reply:
updated, got too used to doing go-style for private